pocs/state/states/default/observing.py

```diff
@@ -13,7 +13,7 @@ def on_enter(event_data):
     pocs.next_state = 'parking'
 
     try:
-        maximum_duration = pocs.observatory.current_observation.exp_time + MAX_EXTRA_TIME
+        maximum_duration = pocs.observatory.current_observation.exp_time.value + MAX_EXTRA_TIME
```

Comment:
How about adding a helper method to the observation that returns numeric seconds (e.g. does a conversion of the time to u.second before extracting .value)?

Reply:
I want to incorporate the idea of a readout time, which makes it seem like it should be that of the camera rather than the observation.

Going to merge this and think about other issue.

Reply:
The non-gphoto2 Camera classes already have a `readout_time` property, though it's currently not used for much.

Reply:
> The non-gphoto2 Camera classes already have a `readout_time` property, though it's currently not used for much.

Yes, that's what I was thinking of. If I add anything it will be an expansion of that existing property.